Expense_Tracker_MVC.csproj

```diff
@@ -13,7 +13,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Expense_Tracker_MVC</RootNamespace>
     <AssemblyName>Expense_Tracker_MVC</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
Can upgrade to .NET 6?

Reply:
Sir, when we update the application, the latest and recommended version for this project is 4.8, as indicated in the attached screenshot.

<img width="419" alt="image" src="https://user-images.githubusercontent.com/98938528/209904825-cf8d03be-5928-4a85-8d82-289228fb5369.png">